<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartDialog</class>
 <widget class="QDialog" name="CartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的购物车</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/icons/cart.png</normaloff>:/new/icons/cart.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	border:0.5px solid white;
	border-radius:6px;
	
	background-color: rgb(233, 255, 224);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:black;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/new/icons/icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	
	background-color:rgb(233, 255, 224);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:15pt;
	font-weight:bold;
	color:blue;
}

</string>
       </property>
       <property name="text">
        <string>我的购物车</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>删除这一项</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	border:0.5px solid black;
	border-radius:6px;
	
	background-color: rgb(233, 255, 224);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:black;
}

</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="saleL">
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="countL">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>共需支付: 不可用</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="payB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>结算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
